contrib/gcppubsub/pkg/dispatcher/dispatcher/reconcile.go

```diff
@@ -312,18 +318,29 @@ func (r *reconciler) receiveMessagesBlocking(ctxWithCancel context.Context, enqu
 	}
 }
 
-func receiveFunc(logger *zap.SugaredLogger, sub pubsubutil.GcpPubSubSubscriptionStatus, defaults provisioners.DispatchDefaults, dispatcher provisioners.Dispatcher) func(context.Context, pubsubutil.PubSubMessage) {
+func receiveFunc(logger *zap.SugaredLogger, sub pubsubutil.GcpPubSubSubscriptionStatus, defaults provisioners.DispatchDefaults, dispatcher provisioners.Dispatcher, rateLimiter *workqueue.RateLimiter) func(context.Context, pubsubutil.PubSubMessage) {
```

Comment:
Why pass the rateLimiter by address? Interfaces are normally passed as-is, rather than as a pointer (because they are normally already pointers to a struct, as in this case).

Reply:
Done! My lack of Go expertise. Thanks for the comment